### ISSUE TITLE
move around games by dragging them

### ISSUE THREAD

**Issue by SousouEnVideo** [JosefNemec/Playnite#3356]
### Feature description

, i want to know if it's ever gonna be considered to be able to just drag and drop games around so you change their positions without having to make categories or sort by name or by series etc, just pick your game and drag it to another position would be very cool

**Comment by Gerren** [comment]
Hi there. I can you where you're aiming, but IMO as long as you have more game, than let's say 50, I can't even imagine to organize all of them one by one. You have to have a system in it, don't you?

But! You *COULD* use the User score property in the Game detail window to set, how high or low it should be on your list.

In THAT case, I could actually imagine, that drag & drop one game between two other would set the User score to the average value: I drag a game between 90 and 80, it would get a 95.

But alas, in no other use case it would not be usable or sustainable to sort by hand.

**Comment by SousouEnVideo** [comment]
That's true, but having the option for small libraries or handpicked, would
be fun, if you can do it it's cool if not it's ok

**Comment by Laerositus** [comment]
Upping this. I'm mostly using categories in my library, and being able to grab a game and update the games info (based on configuration settings) is in my opinion as very nice feature, that Steam already has implemented. 

For example, having categories and dragging a game into a category list could add that category to the game info (or remove it from this category, like I said based on configuration). Dragging a game title to another Completion Status could update it to that status.
